apache/trafficcontrol
# Add TO-Go /api/1.1/servers/status GET handler

## What does this PR (Pull Request) do?
Rewrite the /api/1.1/servers/status GET handler to Go.

- [x] This PR fixes #3826 


## Which Traffic Control components are affected by this PR?
- Traffic Ops


## What is the best way to verify this PR?
1. Run to the TO API tests, make sure they pass.
2. Build the docs, make sure its successful, and view the output to make sure the new table looks good.

## The following criteria are ALL met by this PR
